blog/_posts/2017-8-25-gpuarrays.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Julia's GPU ecosystem: A status repot"
+author: <a href="https://github.com/SimonDanisch">Simon Danisch</a>
+---
+
+A lot has been happening lately in Julia's GPU ecosystem.
+
+[Tim Maleadt](https://github.com/maleadt) has put great effort into making [CUDAnative.jl](https://github.com/JuliaGPU/CUDAnative.jl/) a fully fledged compiler backend to execute Julia code on nvidia GPUs.
```

Comment:
That's not my name...

Reply:
haha, jeez, your Github name has been tricking me for ages :D